buildbot-conf/cordova-config.json.sample

```diff
@@ -0,0 +1,45 @@
+{
+    "couchdb": {
+        "uri":  "http://localhost:5984",
+        "host": "localhost",
+        "port": 5984
```

Comment:
I haven't found a place where "host" or "port" used. Seems it some debug / testing code.